### PR TITLE
fix: journeys card layout

### PR DESCRIPTION
**Pull request by storyworks**
# Description

Includes changes to card layout, conductor layout and adds pagination.

### <samp>🤖 Generated by Copilot at 81fac22</samp>

The pull request improves the card preview components for both the journeys app and the journeys admin app. It refactors the `Card` component and its subcomponents to use styled components, hooks, custom components, and journey theme settings. It also adds support for different cover types, responsive design, and Swiper pagination. It removes some unused components and files, and updates the test cases and styles accordingly.

- Link to Basecamp Todo

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

See https://docs.google.com/spreadsheets/d/1Z6eZIt2rcyee2Zxb3xkBpOzNJSydJ745YC8dROQ62Iw/edit#gid=0
- [ ] Section: Info icon should pull up the options
- [ ] Section: Embedded Journey
- [ ] Section: Right to Left Journey
- [ ] Card appears on admin

Designs
https://www.figma.com/file/Rt5hcy8zJIUwz4SmnfniSG/%E2%9C%85-NS---Editor-(Truth)?type=design&node-id=17031-151468&t=5REcpSPxZiavHAsK-0

# Walkthrough

### <samp>🤖 Generated by Copilot at 81fac22</samp>

*  Add `borderRadius` property to `Box` component to make card preview have rounded corners ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-f46f261ecacd86ac3e40df1941722bffdfb47ccc07eef05e4d08368207685f4aL296-R296))
*  Import `Stack` component from `@mui/material/Stack` to use as a layout component for card preview ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-6ff5d025d21d9d5f7a45ad997c4f141c26398c350dd91a8af333038eabf3cc25R2))
*  Import `StepHeader` component from `@core/journeys/ui/StepHeader` to display step title and description in card preview ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-6ff5d025d21d9d5f7a45ad997c4f141c26398c350dd91a8af333038eabf3cc25R9))
*  Adjust `paddingY`, `height`, `outline`, `outlineOffset`, and `transform` properties of `Box` component to change size, style, and scaling effect of card preview ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-6ff5d025d21d9d5f7a45ad997c4f141c26398c350dd91a8af333038eabf3cc25L73-R75), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-6ff5d025d21d9d5f7a45ad997c4f141c26398c350dd91a8af333038eabf3cc25L132-R145))
*  Change `width` and `height` properties of `SwiperSlide` component to match card preview dimensions ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-6ff5d025d21d9d5f7a45ad997c4f141c26398c350dd91a8af333038eabf3cc25L158-R164))
*  Wrap `Fade` component with `Stack` component to center content vertically and add `StepHeader` component to render step title and description ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-6ff5d025d21d9d5f7a45ad997c4f141c26398c350dd91a8af333038eabf3cc25L170-R180), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-6ff5d025d21d9d5f7a45ad997c4f141c26398c350dd91a8af333038eabf3cc25L185-R195))
*  Change `borderRadius` property of `Box` component to match card preview design and add `my` property to override margin of `SwiperSlide` component ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-e3b621a4c4559744ebece26647642aa64ebf638fa0b134faefcb2225c1f23235L95-R96))
*  Import `swiper/css/pagination` instead of `swiper/components/pagination/pagination.min.css` to use latest version of Swiper pagination styles ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-22793713f9b8530366038063515a6716ee9073a6e2716092e37552c3bbde5d44L14-R14))
*  Import `Pagination` module from `swiper` to enable Swiper pagination feature for card preview ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a8d7730e579396800e245cffe8231010bcec4f0f60fe7ae229fe721377a88f0aL2-R2))
*  Remove `useBreakpoints` hook, `Footer` component, `edgeSlideWidth` and `getResponsiveGap` variables, `gapBetweenSlides` and `setGapBetween` state variables, `windowWidth` and `setWindowWidth` state variables, and `useEffect` hook that updated window width on resize as they were not needed for card preview layout ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a8d7730e579396800e245cffe8231010bcec4f0f60fe7ae229fe721377a88f0aL14-R18), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a8d7730e579396800e245cffe8231010bcec4f0f60fe7ae229fe721377a88f0aL27-R28), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a8d7730e579396800e245cffe8231010bcec4f0f60fe7ae229fe721377a88f0aR37-R75), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a8d7730e579396800e245cffe8231010bcec4f0f60fe7ae229fe721377a88f0aL54))
*  Import `StyledSwiperContainer` and `StyledSwiperSlide` components from `./styles` to use custom styles for Swiper container and slide elements and add `modules` and `pagination` props to `Swiper` component to enable and configure pagination options ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a8d7730e579396800e245cffe8231010bcec4f0f60fe7ae229fe721377a88f0aL115-L140))
*  Remove `icon` prop from `Navigation` component as it was not needed for card preview layout and change `sx` prop to remove display property for icon ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a8d7730e579396800e245cffe8231010bcec4f0f60fe7ae229fe721377a88f0aL154-R166))
*  Remove `alignSx` variable and adjust `top`, `bottom`, `width`, `height`, and `display` properties of `Navigation` components to change position, size, and visibility of navigation buttons ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a8d7730e579396800e245cffe8231010bcec4f0f60fe7ae229fe721377a88f0aL166-R182))
*  Change `color` property of `Navigation` components to use primary contrast text color and add `&:hover` pseudo-class to prevent color change on hover ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a8d7730e579396800e245cffe8231010bcec4f0f60fe7ae229fe721377a88f0aL178-R194))
*  Remove `Box` component that wrapped `Swiper` component and `showLeftButton` and `showRightButton` variables as they were redundant and move `Navigation` components inside `Swiper` component to align them with slides ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a8d7730e579396800e245cffe8231010bcec4f0f60fe7ae229fe721377a88f0aL191-R242))
*  Change `borderRadius` property of `Box` component to use responsive value that matches card preview design and add `overflow` property to hide overflow of embedded preview content ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-d8151a4b652167cad738062ac4194e3b4b2afd6e0e64add15e70c7f51d2a6a73L113-R114))
*  Remove `borderRadius` property from `Box` and `video` components as they were not needed for video wrapper component ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-28d718ce6061eca77c98351c589323f20ddc9d068fbdb8d0780e734870fc6a32L65), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-28d718ce6061eca77c98351c589323f20ddc9d068fbdb8d0780e734870fc6a32L124))
*  Add `BackgroundVideo` component to render video as a background for card preview ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-0a0e9b9125354c7b6e62f19f0aa95209d447de9566b4c86e70c5ade50b2ba6c8R1-R166), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-7fe03f35726fe2373e6376ff32763afae6ecff3d40414a4499a308fde655b657R1))
*  Change `getByTestId` query from `expandedBlurBackground` to `ExpandedImageCover` and remove `toHaveStyle` assertion as it was not relevant for image cover ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-031dbee4834f89ee19bea387a9766835e8c77e6075937db6eeb8aad8d6890ceeL180-R180))
*  Change `getByTestId` query from `ContainedCardImageCover` to `background-image` and change `toHaveAccessibleName` assertion to use image block alt instead of hardcoded string ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-031dbee4834f89ee19bea387a9766835e8c77e6075937db6eeb8aad8d6890ceeL196-R196))
*  Remove `imageBlock` prop from `ContainedCover` component and add `posterBlockId` prop to `videoBlock` prop to use custom poster image for video cover ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-031dbee4834f89ee19bea387a9766835e8c77e6075937db6eeb8aad8d6890ceeL212-R214))
*  Move `videoBlock` and `imageBlock` props from `Card` component to `ContainedCover` component to determine type and source of cover block and add `rtl`, `locale`, and `title` props to `CardWrapper` component to pass direction, language, and title of journey to theme provider ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-83b79888786fd47915f01e3ae6fe2d2fb314332f9ef05f5fa8168dfff174f6cfL36-L56), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-83b79888786fd47915f01e3ae6fe2d2fb314332f9ef05f5fa8168dfff174f6cfL69-R67))
*  Add `rtl`, `locale`, and `title` props to `ThemeProvider` component to set direction, language, and title of journey theme ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-83b79888786fd47915f01e3ae6fe2d2fb314332f9ef05f5fa8168dfff174f6cfR80-R82), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-83b79888786fd47915f01e3ae6fe2d2fb314332f9ef05f5fa8168dfff174f6cfL147-R160))
*  Add `backgroundColor`, `imageBlock`, and `isVideoOnlyCard` props to `ExpandedCover` component to pass background color, image source, and video flag to component and remove `backgroundBlur` prop as it was not needed for expanded cover ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-83b79888786fd47915f01e3ae6fe2d2fb314332f9ef05f5fa8168dfff174f6cfL98-R101))
*  Import `TreeBlock` and `ImageFields` types to use as types for image block prop ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a5ff444e4c3cca4c36c8faf5aa63366c984b8d4e8a1432560ce951b9864d8ffcR2-R3), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-09670d34ddfcd569913a91698252224b5d0dca75ac863303c0d87d187355ed05L2-R81))
*  Add `imageBlock` prop to `ExpandedCover` component to pass mock image block and add `toHaveAccessibleName` assertion to check alt text of image cover ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a5ff444e4c3cca4c36c8faf5aa63366c984b8d4e8a1432560ce951b9864d8ffcR12-R24), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a5ff444e4c3cca4c36c8faf5aa63366c984b8d4e8a1432560ce951b9864d8ffcL15-R41))
*  Remove `ReactNode`, `useEffect`, `useRef`, `useState`, and `CSSProperties` imports as they were not needed for `ContainedCover` component and import `styled`, `Stack`, `useJourney`, `getJourneyRTL`, `BackgroundVideo`, and `OverlayContent` to use styled components feature, layout component, journey data, journey direction, video background component, and card content component with gradient overlay ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-de466867478c6ebdafd8654eee49297476e8dd17f7c99bc8ed3b552c1e161d70L1-R13))
*  Change `children` prop from single element to array of elements to allow multiple children to be rendered and add `contentHeight` and `setContentHeight` state variables to store height of content overlay and `journey`, `rtl`, `contentRef`, `posterImage`, `overlayGradient`, `overlayImageMask`, `StyledGradientBackground`, `StyledSoftBlurBackground`, and `StyledBlurBackground` variables to use journey data, direction, ref, poster image url, gradient function, image mask, and custom styles ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-de466867478c6ebdafd8654eee49297476e8dd17f7c99bc8ed3b552c1e161d70L26-R53), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-de466867478c6ebdafd8654eee49297476e8dd17f7c99bc8ed3b552c1e161d70L40-R94))
*  Replace `Box` and `video` components with `BackgroundVideo` component to render video as a background and wrap `NextImage` component with `Stack` component to apply image mask and replace `ContentOverlay` component with `Stack` and `OverlayContent` components to render overlay gradient and blur elements and add `StyledGradientBackground` component to render overlay gradient for video only card case ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-de466867478c6ebdafd8654eee49297476e8dd17f7c99bc8ed3b552c1e161d70L120-R275))
*  Remove `ReactNode` import as it was not needed for `ExpandedCover` component and import `Stack`, `NextImage`, `useJourney`, `getJourneyRTL`, `BackgroundVideo`, and `OverlayContent` to use layout component, background image component, journey data, journey direction, video background component, and card content component with gradient overlay ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-09670d34ddfcd569913a91698252224b5d0dca75ac863303c0d87d187355ed05L2-R81))
*  Delete `Footer` component and related files as it was not part of card preview ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-840a87bd361611f66dca253081ca2ca852d366126af73399f8d64b61d7c6dfe5), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-84efe9a94ad9ed8cc770e5a6c2ac7c7acd7dc24fbf9f844221b2f66e9ca7b70b), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-2d84e412766ad8b6e36a8a5441a9fd63caec0caefecb30790132243f26706ba7))
*  Delete `ContentOverlay` component and related files as it was replaced by `OverlayContent` component ([link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-148ad49e733d48846bacc7854dea902798b0184030201e0684c2f5c76923ea67), [link](https://github.com/JesusFilm/core/pull/1582/files?diff=unified&w=0#diff-a3180858df038ad3b7da36aa504df653f0de9d4949de0b45ebb89ee03bf763f7))

### CODE REVIEW

File: apps/journeys/src/components/Conductor/Conductor.tsx [hunk 1]
```diff
@@ -112,32 +150,6 @@ export function Conductor({ blocks }: ConductorProps): ReactElement {
     if (activeBlock != null && !activeBlock.locked) nextActiveBlock()
   }
 
-  const [windowWidth, setWindowWidth] = useState(theme.breakpoints.values.xl)
```
Comment (storyworks) [inline apps/journeys/src/components/Conductor/Conductor.tsx:115]: No longer need to calculate dynamic gaps between cards

File: apps/journeys/src/components/Conductor/Conductor.tsx [hunk 2]
```diff
@@ -163,19 +174,24 @@ export function Conductor({ blocks }: ConductorProps): ReactElement {
...
+          height: '80%'
         }}
       >
         <IconButton
           data-testid={`conductor${variant}Button`}
           onClick={handleNext}
           disabled={variant === 'Left' ? disableLeftButton : disableRightButton}
           disableRipple
-          sx={{ color: 'text.primary', px: 13 }}
+          sx={{
```
Comment (storyworks) [inline apps/journeys/src/components/Conductor/Conductor.tsx:190]: Arrow colours are kept roughly same for this PR, will be replaced soon with new arrows / behaviour in next PR.

File: apps/journeys-admin/src/components/Editor/Canvas/Canvas.tsx [hunk 3]
```diff
@@ -155,7 +161,7 @@ export function Canvas(): ReactElement {
                       step.id === selectedStep?.id ? 'none' : 'auto'
                   }}
                 />
-                <FramePortal width={356} height={536} dir={rtl ? 'rtl' : 'ltr'}>
+                <FramePortal width={360} height={640} dir={rtl ? 'rtl' : 'ltr'}>
```
Comment (storyworks) [inline apps/journeys-admin/src/components/Editor/Canvas/Canvas.tsx:164]: Updating to our minimum supported screen size

File: libs/journeys/ui/src/components/Card/BackgroundVideo/BackgroundVideo.tsx [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+        muted: true,
+        loop: true,
+        responsive: true
+        // Don't use poster prop as image isn't optimised
+      })
+      playerRef.current.src(videoSrc)
+    }
+  }, [videoSrc])
+
+  // Set up video listeners and source
```
Comment (storyworks) [inline libs/journeys/ui/src/components/Card/BackgroundVideo/BackgroundVideo.tsx:84]: Oops will update comment to remove "and source"

File: libs/journeys/ui/src/components/Card/BackgroundVideo/BackgroundVideo.tsx [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+          type: 'application/x-mpegURL'
+        }
+
+      default:
+        return ''
+    }
+  }, [source, startAt, endAt, videoId, video?.variant])
+
+  // Initiate Video
+  useEffect(() => {
```
Comment (storyworks) [inline libs/journeys/ui/src/components/Card/BackgroundVideo/BackgroundVideo.tsx:61]: Extracted from ContainedCover:

- Made autoplay true,
- initiated source at beginning
- refactored setup of event listeners to remove nullish playerRef.current check. I initially rewrote this similar to VideoPlayer in watch but reverted in this PR to keep changes to a minimum.

File: apps/journeys/src/components/Footer/Footer.tsx [hunk 6]
```diff
@@ -1,35 +0,0 @@
...
-import Stack from '@mui/material/Stack'
-import Typography from '@mui/material/Typography'
-import MuiLink from '@mui/material/Link'
-import { useTranslation } from 'react-i18next'
-
-export function Footer(): ReactElement {
-  const { t } = useTranslation('apps-journeys')
-
-  return (
-    <Stack direction="row" spacing={3} alignItems="center">
```
Comment (storyworks) [inline apps/journeys/src/components/Footer/Footer.tsx:11]: Content now added to StepHeader

File: libs/journeys/ui/src/components/Card/ContainedCover/ContainedCover.spec.tsx [hunk 7]
```diff
@@ -97,60 +106,83 @@ describe('ContainedCover', () => {
...
 
-    const sourceTag =
-      getByTestId('ContainedCover').querySelector('.vjs-tech source')
-    expect(sourceTag?.getAttribute('src')).toEqual(
-      'https://arc.gt/hls/2_0-FallingPlates/529'
-    )
-    expect(sourceTag?.getAttribute('type')).toEqual('application/x-mpegURL')
-    expect(queryByTestId('VideoPosterCover')).not.toBeInTheDocument()
-    expect(getAllByText('How did we get here?')[0]).toBeInTheDocument()
+    // await waitFor(() => {
```
Comment (storyworks) [inline libs/journeys/ui/src/components/Card/ContainedCover/ContainedCover.spec.tsx:131]: Videos work in storybook / manually when tested but having trouble getting it to show here.. I know src is on the .vjs-tech element and this code queries it's parent, but that element doesn't exist and we get a "this media is not supported error" in jest.

File: libs/journeys/ui/src/components/Card/BackgroundVideo/BackgroundVideo.tsx (hunk 5 above)
Comment (storyworks) [reply libs/journeys/ui/src/components/Card/BackgroundVideo/BackgroundVideo.tsx:33]: This is tested within ContainedCover for now.

File: libs/journeys/ui/src/components/StepHeader/StepHeader.tsx [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+import { useTranslation } from 'react-i18next'
+import Link from '@mui/material/Link'
+import { TreeBlock } from '../../libs/block'
+import { useJourney } from '../../libs/JourneyProvider'
+
+interface StepHeaderProps {
+  block: TreeBlock
+}
+
+export default function StepHeader({ block }: StepHeaderProps): ReactElement {
```
Comment (tataihono) [inline libs/journeys/ui/src/components/StepHeader/StepHeader.tsx:21]: please change the format of these to not export default.

File: libs/journeys/ui/src/components/StepHeader/StepHeader.tsx [hunk 9]
```diff
@@ -0,0 +1,130 @@
...
+      ? block.children[0]
+      : undefined
+
+  const cardTheme = {
+    themeName: card?.themeName ?? journey?.themeName ?? ThemeName.base,
+    themeMode: card?.themeMode ?? journey?.themeMode ?? ThemeMode.dark
+  }
+
+  return (
+    <ThemeProvider {...cardTheme}>
```
Comment (tataihono) [inline libs/journeys/ui/src/components/StepHeader/StepHeader.tsx:45]: should this be extracted to be handled for both the step header and the block renderer?

**Review by tataihono — CHANGES_REQUESTED** [review]

File: libs/shared/ui/src/components/CustomIcon/outlined/InfoCircleContained.tsx [hunk 10]
```diff
@@ -0,0 +1,10 @@
+import { createSvgIcon } from '@mui/material/utils'
+
+export default createSvgIcon(
+  <path
+    fillRule="evenodd"
+    clipRule="evenodd"
+    d="M11 2.4a8.6 8.6 0 1 0 0 17.2 8.6 8.6 0 0 0 0-17.2ZM.4 11a10.6 10.6 0 1 1 21.2 0A10.6 10.6 0 0 1 .4 11ZM11 6.4c.6 0 1 .4 1 1a1 1 0 1 1-2 0c0-.6.4-1 1-1Zm0 3.6c.6 0 1 .4 1 1v4.8a1 1 0 1 1-2 0V11c0-.6.4-1 1-1Z"
+  />,
+  'InfoCircleContained'
+)
```
[the remark this answers is not in the document]
Comment (storyworks) [reply libs/shared/ui/src/components/CustomIcon/outlined/InfoCircleContained.tsx:10]: Stratis

File: apps/journeys-admin/src/components/Editor/Canvas/Canvas.tsx [hunk 11]
```diff
@@ -129,14 +131,18 @@ export function Canvas(): ReactElement {
...
+                      ? step.id === selectedBlock?.id
+                        ? `2px solid ${theme.palette.primary.main}`
+                        : `2px solid ${theme.palette.background.default}`
+                      : `0px solid`,
+                  outlineOffset: 4,
                   transform:
-                    step.id === selectedStep?.id ? 'scaleY(1)' : 'scaleY(0.9)',
-                  height: 536
+                    step.id === selectedStep?.id
+                      ? 'scale(0.95)'
```
[the remark this answers is not in the document]
Comment (storyworks) [reply apps/journeys-admin/src/components/Editor/Canvas/Canvas.tsx:143]: Margins cut it off when scale is 1, 
image

 it seemed close enough + the aspect ratio is correct.

File: libs/journeys/ui/src/components/StepHeader/StepHeader.tsx (hunk 9 above)
Comment (storyworks) [reply libs/journeys/ui/src/components/StepHeader/StepHeader.tsx:45]: Can do.

File: libs/shared/ui/src/libs/themes/base/tokens/components.ts [hunk 12]
```diff
@@ -98,7 +98,7 @@ export const baseComponents: Required<Pick<ThemeOptions, 'components'>> = {
           '&:-webkit-autofill': {
             transitionDelay: '9999s',
             transitionProperty: 'background-color, color',
-            '-webkit-box-shadow': 'none'
```
[the remark this answers is not in the document]
Comment (storyworks) [reply libs/shared/ui/src/libs/themes/base/tokens/components.ts:101]: It was causing a console warning.

File: libs/shared/ui/src/components/CustomIcon/outlined/InfoCircleContained.tsx (hunk 10 above)
[the remark this answers is not in the document]
Comment (tataihono) [reply libs/shared/ui/src/components/CustomIcon/outlined/InfoCircleContained.tsx:10]: please remove until we are advised how to move on with attribution for these icons.

File: libs/journeys/ui/src/components/StepHeader/StepHeader.tsx [hunk 13]
```diff
@@ -1,8 +1,7 @@
 import { ReactElement, useState, MouseEvent } from 'react'
 import Menu from '@mui/material/Menu'
 import MuiMenuItem from '@mui/material/MenuItem'
-import { ThemeProvider } from '@core/shared/ui/ThemeProvider'
-import { ThemeName, ThemeMode } from '@core/shared/ui/themes'
+
```
Comment (tataihono) [inline libs/journeys/ui/src/components/StepHeader/StepHeader.tsx:4]: no space

File: libs/journeys/ui/src/libs/getStepTheme/getStepTheme.ts [hunk 14]
```diff
@@ -0,0 +1,22 @@
...
+export function getStepTheme(
+  block: TreeBlock<StepFields> | TreeBlock<CardFields>,
+  journey?: Pick<JourneyFields, 'themeName' | 'themeMode'>
+): { themeName: ThemeName; themeMode: ThemeMode } {
+  const cardFromStep =
+    block.children.length > 0 && block.children[0].__typename === 'CardBlock'
+      ? block.children[0]
+      : undefined
+
+  const card = block.__typename === 'StepBlock' ? cardFromStep : block
```
Comment (tataihono) [inline libs/journeys/ui/src/libs/getStepTheme/getStepTheme.ts:16]: does this work?
```suggestion
  const card =
    block.__typename === 'StepBlock' && block.children.length > 0 && block.children[0].__typename === 'CardBlock'
      ? block.children[0]
      : block
```

**Review by tataihono — CHANGES_REQUESTED** [review]

File: libs/journeys/ui/src/components/Card/OverlayContent/OverlayContent.tsx [hunk 15]
```diff
@@ -0,0 +1,47 @@
+import { ReactElement, ReactNode } from 'react'
+import { SxProps } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+
+interface OverlayContentProps {
+  children: ReactNode
+  sx: SxProps
+}
+
+export default function OverlayContent({
```
Comment (tataihono) [inline libs/journeys/ui/src/components/Card/OverlayContent/OverlayContent.tsx:10]: change

**Review by tataihono — DISMISSED** [review]

**Review by GeronimoJohn — APPROVED** [review]

**Review by GeronimoJohn — APPROVED** [review]

**Review by GeronimoJohn — APPROVED** [review]

**Review by Kneesal — DISMISSED** [review]

**Review by Kneesal — APPROVED** [review]

**Pull request merged**